docs/configuration.md

```diff
@@ -1136,6 +1136,12 @@ combination with PAPERLESS_CONSUMER_BARCODE_UPSCALE bigger than 1.0.
 
     Defaults to "300"
 
+## Audit Trail
+
+#### [`PAPERLESS_AUDIT_ENABLED=<bool>`](#PAPERLESS_AUDIT_ENABLED){PAPERLESS_AUDIT_ENABLED}
+
+: Enables an audit trail for each document,document type, correspondent, and tag. This records all changes made to the documents to meet requirement such as GoDB. **Once enabled cannot be disabled!!**
```

Comment:
Using a mkdocs [admonition](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) (probably a [warning](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#+type:warning) type) will help it stand out